cluster/kubectl.sh

```diff
@@ -32,15 +32,17 @@ set -o pipefail
 
 KUBE_ROOT=${KUBE_ROOT:-$(dirname "${BASH_SOURCE[0]}")/..}
 source "${KUBE_ROOT}/cluster/kube-util.sh"
-source "${KUBE_ROOT}/cluster/clientbin.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
 
 # If KUBECTL_PATH isn't set, gather up the list of likely places and use ls
 # to find the latest one.
 if [[ -z "${KUBECTL_PATH:-}" ]]; then
-  kubectl=$( get_bin "kubectl" "cmd/kubectl" )
+  kubectl=$( kube::util::find-binary "kubectl" )
 
   if [[ ! -x "$kubectl" ]]; then
-    print_error "kubectl"
+    {
+      echo "It looks as if you don't have a compiled kubectl binary"
```

Comment:
maybe inline the previous error message to match?

Reply:
this the old message:
https://github.com/kubernetes/kubernetes/blob/master/cluster/clientbin.sh#L98-L109

should the whole message be added?

Reply:
I feel like we should continue printing the whole / same message

Reply:
updated.